install/kubernetes/cilium/templates/cilium-agent/peer-service.yaml

```diff
@@ -0,0 +1,24 @@
+{{- if and .Values.hubble.enabled .Values.hubble.listenAddress .Values.hubble.peerService.enabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: hubble-peer
+  namespace: {{ .Release.Namespace }}
+  labels:
+    k8s-app: cilium
+spec:
+  selector:
+    k8s-app: cilium
+  ports:
+  - name: peer-service
+    {{- if .Values.hubble.peerService.servicePort }}
+    port: {{ .Values.hubble.peerService.servicePort }}
+    {{- else }}
+    port: {{ .Values.hubble.tls.enabled | ternary 443 80 }}
+    {{- end }}
+    protocol: TCP
+    targetPort: {{ splitList ":" .Values.hubble.listenAddress | last }}
+{{- if semverCompare ">=1.22-0" .Capabilities.KubeVersion.GitVersion }}
+  internalTrafficPolicy: Local
+{{- end }}
+{{- end }}
```

Comment:
Why is this file in cilium-agent shouldn't this be in hubble-relay?

Reply:
No, it's a service on cilium.

Reply:
But all of its configuration comes from `hubble` flags :thinking:

Reply:
I'm inclined to agree with @aanm here, the `hubble-peer` svc is akin to [`hubble-metrics`](https://github.com/cilium/cilium/blob/641654b2e8273294c6d46714f959f74b820755a7/install/kubernetes/cilium/templates/hubble/metrics-service.yaml#L1): even though its selector is `k8s-app=cilium`, its labels should be `k8s-app=hubble` and be moved to `install/kubernetes/cilium/templates/hubble`. Does that make sense?